### ISSUE TITLE
Implement Inactive List sorting - Alphabetical

### ISSUE THREAD

**Issue by quianalevitt** [RimSort/RimSort#131]
- [x] Alphabetically sort inactive mod list when cache refresh completes
- [x] Alphabetically sort inactive mod list when "Sort" action is completed
- [x] Alphabetically sort inactive mod list when mod list is imported 

Recommendation: When sorting currently by say, Top type, the inactive list is ALSO sorted by this type of sort.  It is likely better to keep the inactive list sort order by only Alphabetical, in order to ensure that the user can find mods by name as expected of a list.

Could also add/allow a sort inactive dropdown for this, if that is easier to allow options/preferences for inactive field.

**Comment by twstagg** [comment]
Copying my comments from Dubs:

twsta: I'm not quite sure what you mean by this, the inactive mods list is never sorted
[10:48 AM]twsta: I suppose if you wanted, we could look into alphabetically sorting that when the sort button is clicked. Otherwise, it is pretty much just a "catch-all" for anything that is not added to an active mod list.

When you click "Refresh" it literally just goes through all available mods and tries to put together the active mod list from your ModsConfig.xml. If any are missing, you can enable the missing mods prompt which will allow you to select variants found in SteamDB, and allow you to download them when any missing mods are found.
[10:49 AM]twsta: At that point, anything that was not added to the active mod list gets dumped to inactive. There really is no rhyme or reason other than the order of which it was parsed.
[10:51 AM]twsta: Similarly, i guess, if you import a mod list. it does a similar thing but it doesn't fully refresh the cache (i think)

**Comment by KarmicBacklash** [comment]
To add to this, having an option to also sort the inactive list in a few different different ways would be appreciated. Off the top of my head I'd like an Alphabetical, date added, last updated, and an option that sorts how old the mod is in general, as in when it was put on the workshop.

**Comment by twstagg** [comment]
I believe this was implemented awhile back.
